pymarsys/contact_list.py

```diff
@@ -0,0 +1,77 @@
+from .base_endpoint import BaseEndpoint
+
+
+class ContactList(BaseEndpoint):
+    """
+    Class representation of the ContactList endpoint.
+
+    Examples:
+    If you want to use contact lists' methods through an instance of the Emarsys
+    client:
+    >>> from pymarsys import SyncConnection, Emarsys
+    >>> connection = SyncConnection('username', 'password')
+    >>> client = Emarsys(connection)
+    >>> client.contact_list
+    <pymarsys.contact_list.ContactList at 0x1050f7048>
+
+    If you want to use contact lists' methods trough an instance of the ContactList
+    endpoint class:
+    >>> from pymarsys import SyncConnection
+    >>> from pymarsys.contact_list import ContactList
+    >>> connection = SyncConnection('username', 'password')
+    >>> contact_list = ContactList(connection)
+    >>> contact_list
+    <pymarsys.contact_list.ContactList at 0x10333ec88>
+    """
+
+    def __init__(self, connection, endpoint="api/v2/contactlist/"):
+        super().__init__(connection, endpoint)
+
+    def create(self, name, key_id=3, with_contacts_id=None, description=None):
+        """
+        Create a contact list.
+        https://dev.emarsys.com/v2/contact-lists/create-a-contact-list
+        :param name: name of the list to create
+        :param key_id: Key which identifies the contact. This can be a field
+        id, id, uid or eid. If left empty, the email address (field ID 3) will
+        be used by default.
+        :param with_contacts: list of key's value to add to the list. e.g. ['squirrel@squirrelmail.com',]
```

Comment:
```suggestion
        :param with_contacts_id: list of key's value to add to the list. e.g. ['squirrel@squirrelmail.com',]
```

Small nitpick, but shouldn't the last fragment of the name be plural, eg `with_contact_ids`?

Reply:
👍